website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,190 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When selecting more than one option is necessary.
+- When list items are displayed in groups.
```

Comment:
HTML select also includes grouping via the [`optgroup` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) so this one isn't much of a reason to use SuperSelect I think...

Reply:
Maybe reword this to say something like:
```suggestion
- When list items are displayed in groups that are visually consistent with our UI.
```

Reply:
Still not positive on the above. @heatherlarsen and @andgen404 do you have any other ideas around the wording?

Reply:
The `dropdown` also supports this, so it should be removed. When possible, we want to document things that help differentiate from other similar components.

Reply:
Removed this line. Thanks for weighing in.